sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -181,6 +185,17 @@ class SampleApplication : Application() {
                 .setService(BuildConfig.APPLICATION_ID)
                 .build()
         )
+        GlobalOpenTelemetry.set(object : OpenTelemetry {
+            override fun getTracerProvider(): OtelTracerProvider {
+                return OtelTracerProvider.Builder()
+                    .setService(BuildConfig.APPLICATION_ID)
```

Comment:
In theory we _can_ remove `setService` call from there and from `AndroidTracer.Builder` above, because by default (if not provided) it will be taken from SDK core configuration, where it is an application name (and if something other was set there it can be a misconfiguration). But we can keep it as an illustration.